lib/runtime/allocator.go

```diff
@@ -56,29 +56,29 @@ func NewAllocator(mem Memory, ptrOffset uint32) *FreeingBumpHeapAllocator {
 		ptrOffset += alignment - padding
 	}
 
-	if mem.Length() <= ptrOffset {
-		err := mem.Grow(((ptrOffset - mem.Length()) / PageSize) + 1)
-		if err != nil {
-			panic(err)
+	if mem.Size() <= ptrOffset {
+		_, ok := mem.Grow(((ptrOffset - mem.Size()) / PageSize) + 1)
+		if !ok {
+			panic("wtf?")
```

Comment:
Can we panic with a more clear message? 😸

Reply:
😂 yes I will address this.

Reply:
nit: wrap the error